packages/components/src/components/carousel/carousel.tsx

```diff
@@ -305,6 +327,7 @@ export class Carousel {
         <div
           id="splide"
           class="splide"
+          {...parseAndGetAriaAttributes(this.aria)}
```

Comment:
Won't the `aria-label` coming from here be overridden by the following line?
I guess a11y tree e2e tests should be added or extended.